debezium-core/pom.xml

```diff
@@ -146,6 +156,23 @@
                     <target>15</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>properties</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <argLine>-javaagent:${io.opentelemetry.javaagent:opentelemetry-agent-for-testing:jar}</argLine>
```

Comment:
Tests have to be run with the opentelemetry java agent to actually test traces emission and TextMapPropagator implemetations behaviors. If not, the tests would go through the default API implementation which is noop, resulting in test failures.

Reply:
@jpechane any concern on this?